web-client/src/App.js

```diff
@@ -37,6 +39,8 @@ function App() {
             <Route path="/signup-reviewer" component={SignUpReviewer} />
             <Route path="/signup-participant" component={SignUpParticipant} />
             <Route path="/scientific-works" component={ScientificWorks} />
+            <Route path="/work-for-review" component={WorkForReview} />
+            <Route path="/my-work" component={MyWork} />
```

Comment:
it should be on the same subpage (we can call it /[WorkTitle] so the content of the website will be different depending on the type of user but URL will be the same)